packages/plugin-assets-retry/src/runtime/index.ts

```diff
@@ -1,3 +1,4 @@
+// rsbuild/initial-chunk/retry
```

Comment:
is is comment necessary

Reply:
Same as webpack runtime code comments, it is easy to debug in browser, debug the anonymous source is painful

https://github.com/web-infra-dev/rsbuild/pull/1978

Reply:
Nice 👍